client/src/styles/layout/_mobile.scss

```diff
@@ -53,3 +53,35 @@ section h2 {
 section ul li {
   @include gap-column-1;
 }
+
+#gameSection {
+  display: flex;
+  flex-direction: row;
+  background-color: #CEF6FF;
+  padding: 20px;
+}
+
+#frame {
+  background-color: lightgrey;
+  border: solid black 2px;
+  margin-top: 20px;
```

Comment:
Man kan skriva in margin med shorthand och i detta fall blir det margin: 20px 0px, men inget måste 🙂